.golangci.yml

```diff
@@ -233,7 +233,7 @@ linters-settings:
       # - filepathJoin
       # - whyNoLint
   gocyclo:
-    min-complexity: 160
+    min-complexity: 165
```

Comment:
Discussed this with @haircommander offline. I'm going to handle the refactoring of `createSandboxContainer` in a separate PR.